src/arbitrary.rs

```diff
@@ -1365,11 +1364,14 @@ mod test {
                 for n in v {
                     let found = shrunk.iter().any(|&i| i == n);
                     if !found {
-                        panic!(format!(
-                            "Element {:?} was not found \
+                        panic!(
+                            "{}",
+                            format!(
+                                "Element {:?} was not found \
                              in shrink results {:?}",
-                            n, shrunk
-                        ));
+                                n, shrunk
+                            )
+                        );
```

Comment:
This change doesn't look right. Why would you use a format-string in a `panic!` just to feed it the output of a `format!`?

Reply:
Yeah I think you can just drop the `format!` call.